src/environmentd/tests/pgwire.rs

```diff
@@ -127,14 +128,62 @@ fn test_bind_params() {
         assert_eq!(vals, &[1, 2]);
     }
 
-    // A `CREATE` statement with parameters should be rejected.
-    let err = client
-        .query_one("CREATE VIEW v AS SELECT $3", &[])
-        .unwrap_db_error();
-    // TODO(benesch): this should be `UNDEFINED_PARAMETER`, but blocked
-    // on database-issues#1031.
-    assert_eq!(err.message(), "there is no parameter $3");
-    assert_eq!(err.code(), &SqlState::INTERNAL_ERROR);
+    // Some statement types can't have parameters at all.
+    //
+    // TODO: these should be `SqlState::UNDEFINED_PARAMETER`.
```

Comment:
I'm planning to submit a quick follow-up PR that fixes this. https://github.com/MaterializeInc/materialize/discussions/5340 is from 2021, and things got better since then.

Reply:
https://github.com/MaterializeInc/materialize/pull/32457